src/app/components/list/view-switcher/index.html

```diff
@@ -106,7 +106,30 @@ <h3>Components</h3>
     >
     </sky-demo-page-module-info>
 
-  </sky-tab>
+    <sky-demo-page-properties
+    sectionHeading="List view switcher custom button properties"
+  >
+    <sky-demo-page-property
+      propertyName="icon"
+      isOptional="true"
+    >
+      Specifies the icon to display in the list view switcher button. This property accepts <sky-code>string</sky-code> properties.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="label"
+      isOptional="true"
+    >
+      Specifies a label to display in the list view switcher button. This property accepts <sky-code>string</sky-code> properties.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="view"
+      isOptional="true"
+    >
+      Specifies the view that the list view switcher button displays. This property accepts <sky-code>ListViewComponent</sky-code> properties.
+    </sky-demo-page-property>
+  </sky-demo-page-properties>
+
+</sky-tab>
```

Comment:
@Blackbaud-AlexKingman, are you planning to create the code samples for this component? Or should I look into trying to set that up? (Unlike the demos, I'm not sure it's really even an option for me to look into creating full code samples, but let me know if I should look into it..)

Reply:
I'm not sure what you mean here by code sample vs demo? Your other question asked about the overall demo so I want to make sure I'm following you correctly. Are you looking for something different than that here?

Reply:
I think I was getting a little ahead of myself. I noticed that we are missing the demo and code sample, so I wanted to make sure both are accounted for. The reason I was thinking of them as distinct is that in the new docs pattern, we have the responsive demo and code samples that expand on different scenarios and aren't just the code for the demo. But we aren't using the new docs pattern here, so my question here didn't make much sense. Sorry.